browser/views.py

```diff
@@ -687,3 +686,50 @@ def delete_investigator_record(request, person_id, research_id):
         Investigator.objects.filter(id=research_id).delete()
 
     return HttpResponseRedirect(reverse('person', args=(person_id,)))
+
+
+@staff_member_required
+def position(request, person_id, position_id=None):
+    person = get_object_or_404(Person, pk=person_id)
+    template = "browser/position.html"
+
+    context = {
+        'person': person,
+        'position_id': position_id
+    }
+
+    if request.method == 'GET':
+        if position_id:
+            position = get_object_or_404(Position, pk=position_id)
+            form = PositionForm(initial={'subject': position.subject, 'role': position.role,
+                                         'year': position.year, 'start_date': position.start_date,
+                                         'end_date': position.end_date})
+            context.update({
+                'form': form
+            })
+        else:
+            form = PositionForm()
+            context.update({
+                'form': form
+            })
```

Comment:
update is duplicate code; move outside of if/else statement.

Reply:
Moved the duplicate code out of the if / else statement